.github/workflows/clang-tidy.yml

```diff
@@ -0,0 +1,28 @@
+name: clang-tidy-review
+on:
+  push:
+    branches: [ master, stable*  ]
```

Comment:
clang-tidy review is meant to run on PRs not pushes.

Reply:
ok, removed.